src/client/src/containers/Footer/index.tsx

```diff
@@ -273,6 +277,9 @@ const mapDispatchToProps = (
   },
   openPostGenModal: () => {
     dispatch(openPostGenModalAction());
+  },
+  updateCreateTemplateButton: () => {
+    dispatch(updateCreateTemplateButtonAction(true));
```

Comment:
I don't think there's a point of this action accepting a boolean if it only ever accepts "true".
Would probably change your code to something like this, 
```ts
    switch(action.type) {
        case WIZARD_INFO_TYPEKEYS.UPDATE_CREATE_TEMPLATE_BUTTON:
            return true;
        case WIZARD_INFO_TYPEKEYS.RESET_WIZARD:
            return false;
        default:
            return state;
    }
```

Reply:
I agree here. However, I feel like keeping the reducer to update whatever boolean value comes with the action.payload is more generic to allow calling the reducer with `false` as the new state. 
That is why, I kept the name to `UPDATE_CREATE_TEMPLATE_BUTTON` rather than `ENABLE_CREATE_TEMPLATE_BUTTON` as this could either flip the state to `true` or `false` depending upon the `action.payload`

Reply:
do you ever plan on making it false again other than after you reset the wizard using that action? Or can you see a potential use case where you would want to?

Reply:
I cannot think of a potential situation right now. Just want to have it since it feels to be more generic.

Reply:
Cool, its not a big deal so i'm fine with it.